client/src/integrationTest/groovy/com/streamr/client/rest/StreamEndpointsSpec.groovy

```diff
@@ -98,6 +98,57 @@ class StreamEndpointsSpec extends Specification {
         getResult.name == createResult.name
     }
 
+    void "createStream() without streamId"() {
```

Comment:
Note that new tests should be written in Unit 5. Refactoring support for projects with two languages is non-existing while Java refactoring support is superb!

Reply:
We'll need to rewrite integration tests as JUnit 5 tests at a later date!

Reply:
Converted the new tests